packages/gasket-plugin-express/index.js

```diff
@@ -42,9 +42,26 @@ module.exports = {
       const compression = require('compression');
 
       const { config } = gasket;
-      const { root, express: { routes } = {} } = config;
+      const { root, express: { routes } = {}, http2 } = config;
       const excludedRoutesRegex = config.express && config.express.excludedRoutesRegex;
-      const app = express();
+      const app = http2 ? require('http2-express-bridge')(express) : express();
```

Comment:
Do we really need this dependency? It seems to be just providing fallback to http1 if alpn is not supported from the middleware https://github.com/rahulramesha/http2-express-bridge/blob/main/src/initMiddleware.js#L15-L16 but otherwise it does a lot of prototype mixing and merging and setting some defaults for the router.

Isn't all of this encapsulated by the `allowHTTP1` option in https://nodejs.org/api/http2.html#http2_http2_createsecureserver_options_onrequesthandler

Reply:
It is a bit more than just that. [This article](https://javascript.plainenglish.io/serving-hello-world-with-http2-and-express-js-4dd0ffe76860) explains the problem well, but `tl;dr` is that Express replaces req and res with custom objects that are incompatible with the http2 object types, which is what this is fixing.

Reply:
TIL express does prototype overwrites 😬